Python/ast.c

```diff
@@ -1477,6 +1560,13 @@ ast_for_arguments(struct compiling *c, const node *n)
                 if (TYPE(ch) == COMMA) {
                     int res = 0;
                     i += 2; /* now follows keyword only arguments */
+
+                    if (i < NCH(n) && TYPE(CHILD(n, i)) == TYPE_COMMENT) {
+                        ast_error(c, CHILD(n, i),
+                                "bare * has associated type comment");
```

Comment:
This looks like an inconsistent indent, should be two more spaces.

Reply:
Fixed, and fixed indents for all other ast_error calls. (I wonder if it used to have a shorter name?)